src/modules/AggregateRepository.js

```diff
@@ -181,5 +202,15 @@ export default class AggregateRepository {
     }
     return new this.EntityClass();
   }
+
+  getEventsByVersion(loadedEvents, version) {
+    let index = loadedEvents.findIndex(({ id }) => {
+      return id === version;
+    });
+    index++;
```

Comment:
The more traditional way to use index is to check it against a condition (`>= 0`) first, and increment it, if used in a loop. However, since it is used only once, there is not really a need to 1) make it a _variable_ (`let`) and not _constant_ (`const`), 2) increment it, 3) preceed condition check with an increment. Thus, I would re-wrote the body of `getEventsByVersion ` like this altogether:

```js
const index = loadedEvents.findIndex(({ id }) => (id === version));
if (index >= 0) {
  return loadedEvents.slice(0, index + 1);
}
```